### DISCUSSION TITLE
Duplicate own_addr from clone in Postgres DB

### DISCUSSION THREAD

**Discussion by Msquared63** [belaban/JGroups#697] [Q&A]
Hi. 

I got the jdbc_ping working.  I cloned the machine, changed the server name, updated it in the keycloak.conf.  I can see in postgres DB I'm getting a duplicate own_addr.

Is there something I can change?  Do I need to create a new node from scratch?

Thanks.

Mike

**Comment by belaban** [comment]
What is this about? You're giving no background on this, and perhaps the keycloak forum would be a better starting place...
